test/phpstan-baseline.neon

```diff
@@ -36,47 +11,17 @@ parameters:
 			path: ../src/Component/PharUpdate/Manifest.php
 
 		-
-			message: "#^Used function Deployer\\\\hostTag not found\\.$#"
+			message: "#^Method Deployer\\\\Component\\\\PharUpdate\\\\Version\\\\Builder\\:\\:clearPreRelease\\(\\) should return Deployer\\\\Component\\\\PharUpdate\\\\Version\\\\Builder but return statement is missing\\.$#"
 			count: 1
-			path: ../src/Component/ProcessRunner/Printer.php
+			path: ../src/Component/PharUpdate/Version/Builder.php
 
 		-
-			message: "#^Constant DEPLOYER_BIN not found\\.$#"
-			count: 2
-			path: ../src/Executor/Master.php
-
-		-
-			message: "#^Variable \\$script in isset\\(\\) is never defined\\.$#"
-			count: 1
-			path: ../src/Importer/Importer.php
-
-		-
-			message: "#^Undefined variable\\: \\$script$#"
-			count: 1
-			path: ../src/Importer/Importer.php
-
-		-
-			message: "#^Variable \\$upload in isset\\(\\) is never defined\\.$#"
-			count: 3
-			path: ../src/Importer/Importer.php
-
-		-
-			message: "#^Undefined variable\\: \\$upload$#"
+			message: "#^Default value of the parameter \\#2 \\$code \\(string\\) of method Deployer\\\\Support\\\\Changelog\\\\ParseException\\:\\:__construct\\(\\) is incompatible with type int\\.$#"
```

Comment:
new errors in the file are from phpstan level 2.

most relevant errors of level 1 were fixed.

level 2 warnings will be fixed in a followup PR.